content/numpy/numpy.md

```diff
@@ -0,0 +1,12 @@
+---
+Title: 'Python:NumPy'
+Description: 'NumPy is a popular open-source Python module, used for complex mathematical operations and multi-dimentional arrays.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/python'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+[NumPy](https://numpy.org/) is a popular open-source Python module used for complex mathematical operations and multi-dimensional arrays. It is a basic library used for most scientific applications. To use NumPy's objects and methods, it will need to be installed first. There are instructions on how to do so on the [NumPy website](https://numpy.org/install/).
```

Comment:
```suggestion
[NumPy](https://numpy.org/) is a popular open-source Python library used for complex mathematical operations and multi-dimensional arrays. It is a basic library used for most scientific applications. To use NumPy's objects and methods, it will need to be installed first. There are instructions on how to do so on the [NumPy website](https://numpy.org/install/).
```